modules/javafx.base/src/test/java/test/javafx/util/converter/LocalDateTimeStringConverterTest.java

```diff
@@ -56,11 +57,9 @@ public class LocalDateTimeStringConverterTest {
 
     private static final DateTimeFormatter aFormatter = DateTimeFormatter.ofPattern("dd MM yyyy HH mm ss");
     private static final DateTimeFormatter aParser = DateTimeFormatter.ofPattern("yyyy MM dd hh mm ss a");
+    private static Locale oldLocale;
```

Comment:
I wonder how many other tests we have that depend on specific Locale?  Perhaps we need to apply the same treatment to:
- LocalDateStringConverterTest
- LocalTimeStringConverterTest

Reply:
I could also change those, as they use `DateTimeFormatter` as well which uses `Locale` underneath as discussed above. @kevinrushforth what do you think?

Reply:
Yes, it seems reasonable to include those tests as well, since those tests have the same problem.